src/components/globals/Sidebar/Sidebar.tsx

```diff
@@ -123,6 +123,8 @@ const Sidebar: React.FC<Props> = props => {
   };
 
   const renderUserRoutes = () => {
+    if (rol !== 1) return null;
```

Comment:
quitar esto, el administrador también debe poder ver estas vistas

Reply:
ahi ponerle if(isAnonymous) return null; el is anonymous lo sacas del Auth() context